common/src/main/java/com/wynntils/features/user/CommandAliasesFeature.java

```diff
@@ -63,35 +63,5 @@ public void onCommandPacket(CommandsPacketEvent event) {
         }
     }
 
-    protected static class CommandAlias {
-        private final String originalCommand;
-        private final List<String> aliases;
-
-        protected CommandAlias(String originalCommand, List<String> aliases) {
-            this.originalCommand = originalCommand;
-            this.aliases = aliases;
-        }
-
-        protected List<String> getAliases() {
-            return aliases;
-        }
-
-        protected String getOriginalCommand() {
-            return originalCommand;
-        }
-
-        @Override
-        public boolean equals(Object other) {
-            if (this == other) return true;
-            if (other == null || getClass() != other.getClass()) return false;
-
-            CommandAlias that = (CommandAlias) other;
-            return originalCommand.equals(that.originalCommand) && aliases.equals(that.aliases);
-        }
-
-        @Override
-        public int hashCode() {
-            return Objects.hash(originalCommand, aliases);
-        }
-    }
+    private record CommandAlias(String originalCommand, List<String> aliases) {}
```

Comment:
Isn't this used in @Config? That means GSON won't work.

Reply:
Ah, I see. Annoying that GSON cannot handle records. :( I'll have to revert it then. Good catch!